src/controller/python/chip-device-ctrl.py

```diff
@@ -639,6 +694,23 @@ def start_rpc_server():
             print("\nKeyboard interrupt received, exiting.")
             sys.exit(0)
 
+def __get_response_dict(status: StatusCodeEnum, result: Optional[Dict[Any, Any]] = None, error:Optional[str] = None) -> Dict [Any, Any]:
```

Comment:
We can narrow down the return type to Dict [str, Any]?

Reply:
Nit: '= None' is not required when marked with Optional AFAIK

Reply:
Resolved.